content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -55,6 +55,7 @@ The example below demonstrates the components of a StatefulSet.
 * The StatefulSet, named web, has a Spec that indicates that 3 replicas of the nginx container will be launched in unique Pods.
 * The volumeClaimTemplates will provide stable storage using [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) provisioned by a PersistentVolume Provisioner.
 
+#### 
```

Comment:
hello @lojies . I am also not in favor of adding an empty heading to address the rendering problem.
The bug should be addressed in Hugo or a Hugo dependency. I am not sure what version of Hugo introduced the bug (again?). I have a local installation of v0.55.6 ext and the page renders OK.
In the meantime, some suggestions to try:
- move the code block before the list and  write a brief caption explaining the contents of the YAML. Move the list after the code block
- indent the start of the code block
- indent the entire code block
- remove the language attribute from the start of the code block
- rewrite the list as a paragraph and test that the code block is formatted correctly

Reply:
+1 @kbhawkey 

It does seem like general consensus is around indenting if you want to maintain a "true" list. It might not be clearly reflected with bullets, but if the list was numbered, this "fix" would break the sequence.

Supporting information [from stackoverflow](https://meta.stackexchange.com/a/7837)

With that, I do see several open issues around blackfriday, which notably hasn't cut a release in over a year.